### PR TITLE
Add blog post from Comcast

### PR DESCRIPTION
**Pull request by posulliv**

### CODE REVIEW

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:39]: For here and all statements .. use uppercase SQL keywords like in the docs

SELECT count(a.c) 
FROM
...

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:48]: add in this sentence what the predicate is..

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:69]: well.. reword .. this is for the more complex version with the predicate right?

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:78]: was to determine possible reasons.

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:80]: link to jstack maybe 

and what is JFR (spell it out)

And 

of the Trino Java process

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:82]: problem. Trino

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:89]: You can leave out the "Notice in the above screenshot"

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:97]: change to present tense

the JVM refuses .. and runs in the much slower interpreted mode.

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:102]: remove "one would expect"

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:118]: Initial results

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:128]: is much more uniform:

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:133]: Code cache

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:136]: few days, and there

File: _posts/2021-10-06-jvm-issues-at-comcast.md
Comment (mosabua) [inline _posts/2021-10-06-jvm-issues-at-comcast.md:202]: Off heap memory usage

**Review by mosabua — CHANGES_REQUESTED** [review]
A bunch of nits and suggestions. 

Beyond that I think you need some sort of Summary at the end.

And also .. can we also get a user testimonial and logo for the site? I can help with details or even send the PR if you supply text, URL and logo. See https://trino.io/users.html
cc @bitsondatadev @martint

**Comment by bitsondatadev** [comment]
@mosabua just so it's clear @posulliv is a support engineer who worked with Comcast but it would be cool if Comcast could add their testimonial. 

Padraig can you speak with David and Saj to see if they would be interested in adding the Comcast logo and testimonial on the Trino site?

**Comment by mosabua** [comment]
I am of course aware of the role of @posulliv ...

**Comment by posulliv** [comment]
@mosabua thank you for the review and many comments! I went through and applied them all. I also added a brief summary at the end.

**Comment by posulliv** [comment]
I'll check with the Comcast folks about a user testimonial on the Trino site and let you know.

**Comment by posulliv** [comment]
@mosabua @bitsondatadev is there anything else you need on my end for this post?

**Comment by mosabua** [comment]
Its up to @martint @dain @electrum and other maintainers to decide on next steps. I can review again if desired.

**Pull request merged**
